earth_enterprise/rpms/build.gradle

```diff
@@ -84,9 +70,26 @@ def stagedInstallDir_server_geplaces =
     new File(stagedInstallDir, 'geplaces/opt/google/share/geplaces')
 def stagedInstallDir_server_user_magic =
     new File(stagedInstallDir, 'server/user_magic')
+def stagedInstallDir_server_version =
+    new File(stagedInstallDir, 'server/opt/google/gehttpd/htdocs/earth/gee_long_version.txt')
 def stagedInstallDir_fusion = new File(stagedInstallDir, 'fusion')
 def packageInstallRootDir = new File('/')
 
+def openGeeVersion =
```

Comment:
Is any of this going to conflict with work in #628 ?

Reply:
I needed this change to test RPM upgrades. If #628 fixes the issue, I'm more than happy to remove my changes

Reply:
I ran out of time last week, so I left it in here. I can split it up if it's deemed necessary.

Reply:
I think there's a ticket that @tst-dsugar may be working on for this purpose.  It may involve a little more than what is here, such as differentiating between 'alpha', 'beta' and 'release' versions, and adding more dependency headers, such as `provides`.